modules/Core/src/main/java/org/terasology/logic/inventory/CharacterInventorySystem.java

```diff
@@ -115,6 +115,7 @@ public void onDropItemRequest(DropItemRequest event, EntityRef character) {
             }
         }
 
+        pickupItem.getComponent(ItemComponent.class).pickUpTime = 2000;
```

Comment:
Although very likely,  there is no guarantee that his will have an ItemComponent present.